lib/jorb/backends/file.ex

```diff
@@ -0,0 +1,53 @@
+defmodule Jorb.Backends.File do
+  @moduledoc ~S"""
+  File-based backend for light queueing.
+
+  You probably don't want to use this for production load.
+  """
+  @behaviour Jorb.Backend
+  @queue_dir Path.join("tmp", "jorb")
+
+  def enqueue(queue_name, payload) do
+    ensure_queue_dir(queue_name)
+    message_name = (:erlang.system_time(:milli_seconds) |> Integer.to_string()) <> ".json"
+
+    # We're double-encoding the body here so we can have
+    # parity with how messages come out of SQS
+    final_payload = %{receipt_handle: message_name, body: Poison.encode!(payload)}
+
+    Path.join(queue_dir(queue_name), message_name)
+    |> File.write(Poison.encode!(final_payload))
+  end
+
+  def dequeue(queue_name) do
+    ensure_queue_dir(queue_name)
+    files = queue_dir(queue_name) |> File.ls!()
+
+    if Enum.empty?(files) do
+      []
+    else
+      files
+      |> List.first()
+      |> String.replace_prefix("", "#{queue_dir(queue_name)}/")
+      |> File.read!()
+      |> Poison.decode!()
+      |> Enum.reduce(%{}, fn {key, val}, acc -> Map.put(acc, String.to_atom(key), val) end)
```

Comment:
Please justify use of `String.to_atom/1` over `String.to_existing_atom/1`.  I'm actually not clear why we're using atom keys in this map at all -- it's externally-sourced data, we can't trust it enough to turn it into atoms in general.

Reply:
it should be `to_existing_atom` but the reason is to have parity with the data structure that we get back from SQS.

The only keys that should be in the outer map (the one that wraps the params) should be `receipt_handle` and `body` and the reason for atomizing them is because Poison always returns string keys